tutorials/kubernetes/install-kubernetes-with-conjure-up.md

```diff
@@ -0,0 +1,160 @@
+---
+id: install-kubernetes-with-conjure-up
+summary: This tutorial will guide you through the installation of the Canonical Distribution of Kubernetes with a few simple commands using conjure-up.
+categories: kubernetes
+status: Published
+feedbacklink: https://github.com/ubuntu/codelabs/issues
+tags: kubernetes, conjure-up
+difficulty: 3
+published: 2017-08-21
+
+---
+
+
+# Install Kubernetes with conjure-up
+
+
+## Overview
+Duration: 1
+
+This tutorial will guide you through the installation of the Canonical Distribution of Kubernetes with a few simple commands using conjure-up.
+
+### The Canonical Distribution of Kubernetes
+
+In partnership with Google, Canonical now delivers a ‘pure K8s’ experience, tested across a wide range of clouds and integrated with modern metrics and monitoring.
+
+The Canonical Distribution of Kubernetes works across all major public clouds and private infrastructure, enabling your teams to operate Kubernetes clusters on demand, anywhere.
+
+### conjure-up
+
+conjure-up lets you summon up a big-software stack as a “spell” — a model of the stack, combined with extra know-how to get you from an installed stack to a fully usable one.
+
+### What you’ll learn
+  - How to install software using a conjure-up spell
+  - The simplest way to install The Canonical Distribution of Kubernetes
+
+### What you’ll need
+  - A Linux system with [snapd installed](https://snapcraft.io/docs/core/install)
+
+#### For cloud deployment
+  - Account credentials for one of the following public cloud providers:
+    - AWS
+    - Azure
+    - CloudSigma
+    - Google
+    - Joyent
+
+#### For local deployment
+
+Although this tutorial will assume a public cloud installation, if you wish to install locally the following minimum specifications are recommended:
+ - 2 cores
+ - 16G RAM
+ - 32G Swap
+ - 250G SSD with a separate block device for ZFS. Our recommendation for that device is 100G.
+
+## Getting started
+Duration: 5
+
+If you have not done so on your local machine, install conjure-up.
+
+On a Linux system with snapd simply type the following:
+
+```bash
+sudo snap install conjure-up --classic
+conjure-up
+```
+
+positive
+: Note: If you need to install snapd on your Linux system, detailed instructions can be found in the [snapcraft.io documentation](https://snapcraft.io/docs/core/install).
+
+Once you have completed the conjure-up installation, run the software:
+
+```bash
+$ conjure-up
```

Comment:
You have already run this command above. If you followed these steps you would have to cancel conjure-up to run it again.